api/src/main/java/com/codeforcommunity/rest/subrouter/ProtectedSiteRouter.java

```diff
@@ -446,19 +444,16 @@ private void handleFilterUnapprovedSiteImages(RoutingContext ctx) {
     JWTData userData = ctx.get("jwt_data");
 
     Optional<Timestamp> submittedStart =
-        RestFunctions.getOptionalQueryParam(ctx, "submittedStart", (date) -> new Timestamp(Date.valueOf(date).getTime()));
+        RestFunctions.getOptionalQueryParam(
+            ctx, "submittedStart", (date) -> new Timestamp(Date.valueOf(date).getTime()));
     Optional<Timestamp> submittedEnd =
-        RestFunctions.getOptionalQueryParam(ctx, "submittedEnd", (date) -> new Timestamp(Date.valueOf(date).getTime()));
-    Optional<List<Integer>> siteIds =
         RestFunctions.getOptionalQueryParam(
-            ctx,
-            "siteIds",
-            this::parseOptionalQueryParamList);
+            ctx, "submittedEnd", (date) -> new Timestamp(Date.valueOf(date).getTime()));
+    Optional<List<Integer>> siteIds =
+        RestFunctions.getOptionalQueryParam(ctx, "siteIds", this::parseOptionalQueryParamList);
```

Comment:
Just to make sure, this is all formatting stuff an no logic change

Reply:
I think this is all from the auto-formatter, yeah